pyanaconda/modules/payloads/base/utils.py

```diff
@@ -75,6 +75,6 @@ def sort_kernel_version_list(kernel_version_list):
 
 def _compare_versions(v1, v2):
     """Compare two version number strings."""
-    first_version = LooseVersion(v1)
-    second_version = LooseVersion(v2)
+    first_version = parse(v1)
+    second_version = parse(v2)
```

Comment:
Don't we want to use `Version` instead of `parse`?

Reply:
I'm not actually really sure.

Here's why I went with this, so you can decide yourself: Here we are, (ab)using some *Python* versioning machinery for *kernel* versions. The former code from `distutils` ran well enough it seems. But `Version` is supposed to follow *Python's* version rules that were defined more accurately later. So I think that's a poor idea. `parse`is supposed to actually dynamically select the right thing, and the material I saw prefers it over instancing the `Version` or `LegacyVersion` directly. I  did not dig too deep for that.

I suppose another possibility is to just go with `LegacyVersion` directly?

Reply:
That makes sense. In that case, I would import `parse` as `parse_version` or call `packaging.version.parse()`  to avoid conflicting names in the future. Otherwise, it is ok.

Reply:
Ok, did that.